src/simulation/MixingModels.h

```diff
@@ -260,7 +260,7 @@ template<typename T>
 class DiffusionMixingModel : public MixingModel<T> {
 
 private:
-    int resolution = 10;
+    int numFourierTerms = 100;
```

Comment:
@micheltakken what do you think about renaming the resolution that defines the number of terms in the fourier series for the analytical solution of diffusion as numFourierTerms

Reply:
This is really picking out the detail, but I think `noFourierTerms` works better, or perhaps `fourierLimit` or `fourierTruncation`. However, feel free to ignore and use `numFourierTerms`, which is in my opinion also clear.

Reply:
Let's go with noFourierTerms